telemetry/actioneventobserver.cpp

```diff
@@ -85,7 +85,7 @@ QPair<QString, QString> ActionEventObserver::extractActionData(QObject* watched)
 
 bool ActionEventObserver::eventFilter(QObject *watched, QEvent *event)
       {
-      if (event->type() == QEvent::MouseButtonRelease) {
+      /*if (event->type() == QEvent::MouseButtonRelease) {
```

Comment:
I'd prefer `#if 0` to disaböe a larger block of code

Reply:
This is a PR for testing the integration with Travis. I didn't even test it :)